extensions/wikia/Search/classes/MediaWikiService.php

```diff
@@ -551,41 +551,41 @@ public function getMainPageUrlForWikiId( $wikiId ) {
 	public function getMainPageIdForWikiId( $wikiId ) {
 		return $this->getMainPageTitleForWikiId( $wikiId )->getArticleId();
 	}
-	
+
 	/**
 	 * Returns text from the main page of a provided wiki.
 	 * @param int $wikiId
 	 * @return string
 	 */
 	public function getMainPageTextForWikiId( $wikiId ) {
 		$params = array(
-				'controller' => 'ArticlesApiController', 
-				'method' => 'getDetails', 
+				'controller' => 'ArticlesApiController',
+				'method' => 'getDetails',
 				'titles' => $this->getMainPageTitleForWikiId( $wikiId )->getDbKey()
 				);
 		$response = (new \ApiService)->foreignCall( $this->getDbNameForWikiId( $wikiId ), $params, \ApiService::WIKIA );
 		$item = \array_shift( $response['items'] );
 		return $item['abstract'];
 	}
-	
+
 	/**
 	 * Returns text from the main page of a provided wiki.
 	 * @param int $wikiId
 	 * @return string
 	 */
 	public function getDescriptionTextForWikiId( $wikiId ) {
 		$response = (new \ApiService)->foreignCall(
-			$this->getDbNameForWikiId( $wikiId ), 
+			$this->getDbNameForWikiId( $wikiId ),
 			array(
 					'action'      => 'query',
 					'meta'        => 'allmessages',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "allmessages" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)